fling/utils/visualize_utils/demo/demo_conv_kernel_visualize.py

```diff
@@ -0,0 +1,14 @@
+from torchvision.models import resnet18
+
+from fling.utils import Logger
+from fling.utils.visualize_utils import plot_conv_kernels
+
+if __name__ == '__main__':
+    # Step 1: prepare the model.
+    model = resnet18(pretrained=True)
```

Comment:
To load a model defined within the fling framework from local, the following code can be use:
```
from easydict import EasyDict
from fling.utils.registry_utils import MODEL_REGISTRY

# Step 1: prepare the model.
model_arg=EasyDict(dict(
    name='resnet8',
    input_channel=3,
    class_number=100,
))
model_name = model_arg.pop('name')
model = MODEL_REGISTRY.build(model_name, **model_arg)
model.load_state_dict(torch.load('./local_model.ckpt'))
```